Documentation/ceph-monitoring.md

```diff
@@ -210,3 +210,30 @@ labels:
     prometheus: k8s
 [...]
 ```
+
+### Horizontal Pod Scaling using Kubernetes Event-driven Autoscaling aka KEDA
+
+Using metrics exported from the Prometheus service, the horizontal pod scaling can use the custom metrics other than CPU and memory consumption. It can be done with help of Promethues Scaler provided by the [KEDA](https://keda.sh/docs/1.4/scalers/prometheus/). The KEDA deployed via helm chart, refer [here](https://keda.sh/docs/2.4/deploy/) for details.
+
+Following is sample for `ScaledObject`for autoscaling RGW:
+```YAML
```

Comment:
Please also create this yaml in the [monitoring examples folder](https://github.com/rook/rook/tree/master/cluster/examples/kubernetes/ceph/monitoring).